enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/impl/service/PresenceCheckService.java

```diff
@@ -205,7 +204,7 @@ public void cleanup(OwnerId ownerId) throws PresenceCheckException, RemoteCommun
      * @throws PresenceCheckException In case of business logic error.
      * @throws RemoteCommunicationException In case of remote communication error.
      */
-    private void initPresentCheckWithImage(final OwnerId ownerId, final IdentityVerificationEntity idVerification)
+     protected void initPresentCheckWithImage(final OwnerId ownerId, final IdentityVerificationEntity idVerification)
```

Comment:
Ok, this is a compromise for testing, but may we test it through the public method `init(OwnerId ownerId, String processId)`?